lib/model.js

```diff
@@ -3747,6 +3747,24 @@ class Model {
   }
 
   /**
+   * Helper method to determine if a instance is "soft deleted".  This is
+   * particularly useful if the implementer renamed the `deletedAt` attribute
+   * to something different.  This method requires `paranoid` to be enabled.
+   *
+   * @returns {Boolean}
+   */
+  isSoftDeleted() {
```

Comment:
Yeah, shouldn't the function just check if `deletedAt` is set and less than current datetime? The default value logic looks weird to me